UsingSimpleSqlProvider/app/src/androidTest/java/me/gurpreetsk/sample/usingsimplesqlprovider/ExampleInstrumentedTest.java

```diff
@@ -0,0 +1,26 @@
+package me.gurpreetsk.sample.usingsimplesqlprovider;
```

Comment:
Package name should follow standards as mentioned in [Pull Request guide](https://github.com/nisrulz/android-examples#pull-requests)